business-materials/01_事業内容一覧_社会的意義.md

```
@@ -2,43 +2,122 @@
+    - 国際交流での言語バリアを解消（多言語対応）
+    - 誤解のないコミュニケーションを促進し、社会的つながりを強化
+- **具体的な価値提供**
+    - スマホをかざすだけで情報共有
+    - 常に最新情報を自動更新
+    - アプリ不要で誰でも使える
+    - 多言語対応で国際ビジネス支援
+    - AI名刺撮影・VCard変換で連絡管理を効率化
+- **料金体系（税込）**
+    - 初期費用: **3,000円**
+    - 2枚目以降: 600円/枚
+    - 100枚以上一括注文時: 500円/枚(2枚目以降)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)